packages/runtime/container-runtime/src/messageTypes.ts

```diff
@@ -147,7 +154,8 @@ export type InboundContainerRuntimeMessage =
 	| ContainerRuntimeAliasMessage
 	| ContainerRuntimeIdAllocationMessage
 	| ContainerRuntimeGCMessage
-	| ContainerRuntimeDocumentSchemaMessage
+	| InboundContainerRuntimeDocumentSchemaMessage
+	| OutboundContainerRuntimeDocumentSchemaMessage
```

Comment:
Just Inbound here

Reply:
Changed in https://github.com/microsoft/FluidFramework/pull/24685